src/wrapper/SerializationWrapper.cpp

```diff
@@ -67,6 +67,70 @@ char fs_serializer_openmode(void* serializer)
     return ' ';
 }
 
+int fs_serializer_metainfo(void* serializer)
+{
+    return reinterpret_cast<const Serializer*>(serializer)->globalMetainfo().size();
+}
+
+void fs_serializer_metainfo_key_lengths(void* serializer, int* lengths)
+{
+    const std::vector<std::string> keys = reinterpret_cast<Serializer*>(serializer)->globalMetainfo().keys();
+    const int N = keys.size();
+    for (int i = 0; i < N; ++i)
+        lengths[i] = keys.at(i).size();
+}
+
+void fs_serializer_metainfo_get_keys(void* serializer, char** keys)
+{
+    const std::vector<std::string> mkeys = reinterpret_cast<Serializer*>(serializer)->globalMetainfo().keys();
+    const int N = mkeys.size();
+
+    for (int i = 0; i < N; ++i)
+    {
+        std::strcpy(keys[i], mkeys.at(i).c_str());
+    }
+}
+
+void fs_serializer_metainfo_get_types(void* serializer, int* types)
+{
+    const std::vector<int> mtypes = reinterpret_cast<Serializer*>(serializer)->globalMetainfo().types();
+    const int N = mtypes.size();
+    for (int i = 0; i < N; ++i)
+        types[i] = mtypes.at(i);
+}
+
+void fs_get_serializer_metainfo_b(void* serializer, const char* name, int name_length, bool* value)
```

Comment:
so we could say it in the doxygen so that the user is aware of the exception (and no default value)?
```
@brief method that retrieves a boolean property from metainfo
@param serializer blabla
@throw in case key is not found
```